pandas/plotting/_core.py

```diff
@@ -2470,8 +2470,11 @@ def hist_series(self, by=None, ax=None, grid=True, xlabelsize=None,
         bin edges are calculated and returned. If bins is a sequence, gives
         bin edges, including left edge of first bin and right edge of last
         bin. In this case, bins is returned unmodified.
-    bins: integer, default 10
-        Number of histogram bins to be used
```

Comment:
This is repeated and seems redundant.

Reply:
Repeated where? This should still stay, no?

Reply:
`bins` is defined on line 2468 and 2473. It looks like a mistake when writing the docstring. What do you think?

Reply:
Ah OK didn't see that. This is fine then